Reword About text to be more understandable by a lay person (#1988)
organicmaps/organicmaps: data/strings/strings.txt

  [about_description]
    comment = Text under the version number in the Help dialog.
    tags = android,ios

>     en = Organic Maps is a free offline maps app, fast and easy to use. No ads. No tracking. Maps are based on crowd-sourced OpenStreetMap data, so you can fix mapping errors and add features there yourself. Organic Maps is an open source project created by enthusiasts in their spare time - your feedback and support are much appreciated!

Reviewer: ```suggestion
    en = Organic Maps is a fast and free maps app without ads and tracking. Maps are based on crowd-sourced OpenStreetMap.org data, so you can fix mapping errors and add features there yourself. Organic Maps is an open-source project created by enthusiasts in their spare time. Your feedback and support are much appreciated!
```

Author: offline потерялось.. точно его убрать хотел?
а ссылки там поддерживаются?
Было бы здорово OSM сделать сразу ссылкой на [Beginner's Guide](https://wiki.openstreetmap.org/wiki/Beginners%27_guide)

Reviewer: Это можно будет чуть позже подпилить, ссылки и в ведре и в айосе должны рендерится и кликаться при минимальном шаманстве. Offline забыл, точно, спасибо.

  [about_description]

>     comment = Text under the version number in the Help dialog.

Reviewer: ```suggestion
    comment = Text under the version number in the Help dialog. TODO: Synchronize other translations with English.
```

    pt = Organic Maps é uma aplicação gratuita e de código aberto de mapas offline. Sem anúncios. Sem seguimento. Se vir um erro no mapa, por favor repare-o em OpenStreetMap. O projecto é criado por entusiastas no nosso tempo livre, por isso precisamos do seu feedback e apoio.
    pt-BR = Organic Maps é uma aplicação gratuita e de código aberto de mapas off-line. Sem anúncios. Sem rastreamento. Se você vir um erro no mapa, por favor, corrija em OpenStreetMap. O projeto é criado por entusiastas em nosso tempo livre, então precisamos de seu feedback e suporte.
    ro = Organic Maps este o aplicație gratuită și cod sursă public care permite descărcarea hărților și navigare fără internet. Fără reclame. Fără urmărire. Dacă vezi o eroare pe hartă, te rugăm să o corectezi în OpenStreetMap. Proiectul este creat de entuziaști în timpul nostru liber, așa că avem nevoie de părerea și sprijinul tău.

>     ru = Organic Maps — бесплатные оффлайн карты, простые и удобные, без рекламы и сбора персональных данных. Карты основаны на данных проекта OpenStreetMap, там Вы можете сами исправлять ошибки и делать дополнения. Organic Maps это проект с открытым исходным кодом, создаваемый энтузиастами в свободное время. Будем рады Вашей поддержке и обратной связи!

Author: Сначала я сделал "быстрые и удобные" потом убрал, т.к. это не очень понятно звучит, кмк. Редко по-русски услышишь "быстрое приложение". Но можно и вернуть.
Простота и удобство это, на мой взгляд, то, что отличает, и куда стремится ОМ, поэтому заявлять это норм.

Лучше предлагай правки целыми предложениями, чтобы связно было.

>     ru = Organic Maps — бесплатные оффлайн карты, простые и удобные, без рекламы и сбора персональных данных. Карты основаны на данных проекта OpenStreetMap, там Вы можете сами исправлять ошибки и делать дополнения. Organic Maps это проект с открытым исходным кодом, создаваемый энтузиастами в свободное время. Будем рады Вашей поддержке и обратной связи!

Reviewer: Мы когда-то уже это проходили в маркетинге. Заявление что приложение простое и удобное, если человек не может найти как сделать лонгтап и построить маршрут, например, может выбесить. Всё в мире относительно, кому-то покажется неудобным и сложным.

А вот по скорости у нас конкурентов нет.

>     ru = Organic Maps — бесплатные оффлайн карты, простые и удобные, без рекламы и сбора персональных данных. Карты основаны на данных проекта OpenStreetMap, там Вы можете сами исправлять ошибки и делать дополнения. Organic Maps это проект с открытым исходным кодом, создаваемый энтузиастами в свободное время. Будем рады Вашей поддержке и обратной связи!

Reviewer: be = Organic Maps — гэта хуткія і бясплатныя мапы без рэкламы, якія працуюць без Інтэрнэту і не шпіёняць за вамі. Усе картаграфічныя дадзеныя бяруцца з OpenStreetMap.org, там вы зможаце самастойна выпраўляць памылкі і дадаваць новыя аб'екты. Organic Maps — гэта праект з адкрытым зыходным кодам, які ствараецца энтузіястамі ў вольны час. Будзем рады вашай падтрымцы і зваротнай сувязі!

>     ru = Organic Maps — бесплатные оффлайн карты, простые и удобные, без рекламы и сбора персональных данных. Карты основаны на данных проекта OpenStreetMap, там Вы можете сами исправлять ошибки и делать дополнения. Organic Maps это проект с открытым исходным кодом, создаваемый энтузиастами в свободное время. Будем рады Вашей поддержке и обратной связи!

Author: Мне только "не шпионят" не очень нравится. Не privacy-concerned people (т.е. большинство) разве что о теории заговора может подумать :) Без сбора персональных данных - гораздо нейтральнее и понятнее, имхо.

>     ru = Organic Maps — бесплатные оффлайн карты, простые и удобные, без рекламы и сбора персональных данных. Карты основаны на данных проекта OpenStreetMap, там Вы можете сами исправлять ошибки и делать дополнения. Organic Maps это проект с открытым исходным кодом, создаваемый энтузиастами в свободное время. Будем рады Вашей поддержке и обратной связи!

Reviewer: Многие даже не понимают, что такое "сбор" и "персональные данные". Зато не шпионят очень ярко выражает суть ) Но я не настаиваю.
